### PR TITLE
Nam_show_data_topalbum

### PR DESCRIPTION
**Pull request by NguyenNam34**

### CODE REVIEW

File: app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumAdapter.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            if (album.length() > LENGTH_NAME) {
+                String supAlbum = album.substring(BEGIN_NAME, LENGTH_NAME);
+                mTxtAlbum.setText(supAlbum);
+            } else {
+                mTxtAlbum.setText(album);
+            }
+        }
+
+        private void setImage(int position) {
+            String imageAlbum = mTopAlbums.getAlbum().get(position).getImage().get(SMALL_IMAGE).getText();
```
Comment (daolq3012) [inline app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumAdapter.java]: ERROR: Line is longer than 100 characters (found 106).

File: app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumFragment.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package com.fstyle.androidtrainning.screen.main.mainfragments.online.subfragment.topalbum;
 
+import android.content.Context;
```
Comment (daolq3012) [inline app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumFragment.java]: ERROR: Unused import: android.content.Context. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

File: app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumFragment.java [hunk 3]
```diff
@@ -37,6 +41,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     }
 
     private void initViews(View v) {
+        mRecyclerView = v.findViewById(R.id.recycler_top_album);
+        GridLayoutManager manager = new GridLayoutManager(getActivity(),TWO_COLUMN);
```
Comment (daolq3012) [inline app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumFragment.java]: ERROR: ',' is not followed by whitespace.

File: app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumFragment.java (hunk 2 above)
Comment (daolq3012) [inline app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/topalbum/TopAlbumFragment.java]: WARNING: [UnusedImports] Avoid unused imports such as 'android.content.Context'
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#UnusedImports

**Comment by tranminhtri911** [comment]
ok

**Pull request merged**